Source/Widgets/Timeline/Timeline.js

```diff
@@ -251,6 +253,11 @@ define([
         }
         //>>includeEnd('debug');
 
+        var timesValid = JulianDate.greaterThanOrEquals(startTime, Iso8601.MINIMUM_VALUE) && JulianDate.lessThanOrEquals(stopTime, Iso8601.MAXIMUM_VALUE);
```

Comment:
Right, so in case someone else calls the `zoomTo` with invalid time arguments we'll just throw and make it clear that the arguments are outside the valid time span. Are there plug-ins or something similar that integrate or could be expected to integrate with the timeline?